staging/src/k8s.io/kubectl/pkg/cmd/completion/completion.go

```diff
@@ -161,3 +165,15 @@ func runCompletionZsh(out io.Writer, boilerPlate string, kubectl *cobra.Command)
 
 	return kubectl.GenZshCompletion(out)
 }
+
+func runCompletionPwsh(out io.Writer, boilerPlate string, kubectl *cobra.Command) error {
+	if len(boilerPlate) == 0 {
+		boilerPlate = defaultBoilerPlate
+	}
+
+	if _, err := out.Write([]byte(boilerPlate)); err != nil {
+		return err
+	}
+
+	return kubectl.GenPowerShellCompletion(out)
```

Comment:
If you want to have descriptions for the completion choices, likes zsh has, you can instead use `GenPowerShellCompletionWithDesc()`.

See https://github.com/spf13/cobra/blob/master/shell_completions.md#powershell-completions

Reply:
Thanks for the comment. I don't typically use the "Menu Complete" completion to see those descriptions, but I'm sure other people will appreciate it. I went ahead and updated the method.